File: old/Peer.java
package project.ds.transaction;

public class Peer {
	private long id;
	private long securityKey;
	private float balance;
	
	public long getId() {
		return id;
	}
	
	public long getSecurityKey() {
		return securityKey;
	}
	
	public float getBalance() {
		return balance;
	}
	
	public void setId(long id) {
		this.id = id;
	}
	
	public void setSecurityKey(long securityKey) {
		this.securityKey = securityKey;
	}
	
	public void setBalance(float balance) {
		this.balance = balance;
	}
	
	@Override
	public String toString() {
		return id + " " + balance + " " + securityKey; 
	}
}

File: old/Peer.py
class Peer(object):
    """This is the PEER object"""

    def __init__(self):
        # Init the object
        self.id = 0;
        self.privateKey = 0;
        #self.balance = 0;

    def getId(self):
        # Return the self id
        return self.id;

    def getPrivateKey(self):
        # Return the self private key
        return self.privateKey;

    '''
    def getBalance(self):
        # Return the self balance
        return self.balance;
    '''

    def setPrivateKey(self, privateKey):
        # Set the private key
        self.privateKey = privateKey;

    '''
    def setBalance(self, balance):
        # Set the balance
        self.balance = balance;
    
    def toString(self):
        # Return the self id with the self private key and the self balance
        return self.id + " " + self.privateKey + " " + self.balance;
    '''
    
 